cmake/find/FindNASM.cmake

```diff
@@ -0,0 +1,35 @@
+# This module will set `NASM_FOUND` if NASM is found.
+# If NASM is found, `NASM_ROOT` will be set to the root folder.
+# `NASM_BIN_DIR` will be set to the folder containing the executable,
+# and `NASM_EXECUTABLE` will be set to the path of the NASM executable.
+
+
+if(HUNTER_STATUS_DEBUG)
+  message("[hunter] Custom FindNASM module")
+endif()
+
+# Try to find it even if it may already be set.
+find_path(
+  NASM_ROOT
+  NAMES
+  bin/nasm
+  bin/nasm.exe
+  )
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(
+  NASM
+  DEFAULT_MSG
+  NASM_ROOT
+  )
+
+if(NASM_FOUND)
```

Comment:
If you use hunter to add it, then of course. But this script is rather independent from hunter. The thing I'm thinking is usually the *nix system provides the `nasm` built by the system maintainer, so there is no need to build hunter by oneself if there is already one.

I'm thinking the usage might be like

```
find_package(nasm)
if(NOT NASM_FOUND)
hunter_add_package(nasm)
find_package(nasm)
endif()
```